### ISSUE TITLE
Feature Requests

### ISSUE THREAD

**Issue by markerikson** [markerikson/joplin-plugin-vscode-style-search#2]
Collected from the initial search thread:

> https://discourse.joplinapp.org/t/search-functionality-in-joplin/35787/74?u=acemarke
> - Can the search results be displayed in a notebook structure?
> - Can the search results display the path where the notes are located?
> - Can the search scope include the path where the note is located, in addition to the note title and body text?

> https://discourse.joplinapp.org/t/search-functionality-in-joplin/35787/77?u=acemarke
> - Can the search bar be focused upon showing the panel?
> - If the above is too invasive, can a keyboard shortcut be added to focus the search bar?
> - Can a plugin settings page option be added to preserve search results when hiding the panel?
> - Can clicking a note's result line scroll to where it's located in the note, like how HTML anchors work?

**Comment by kacnje** [comment]
Agree. Focus to the search bar is something I was missing too. And maybe a simpler keyboard shortcut.

**Comment by markerikson** [comment]
Yeah, focus is pretty easy to add. FWIW `CTRL-SHIFT-F` is the standard shortcut that activates full text search in VS Code, so that's why I grabbed it.  What would you like to see that's "simpler"?

**Comment by kacnje** [comment]
I understand. I don't use VS Code and it was the first time to meet this keyboard shortcut. But don't worry, maybe it is just strange for me (using three keys which are all on the left side). If this shortcut can be adjustable and if the focus can be in the search field ...

I'm sure many will use this plugin because searching is Joplin's problem number 1 for many.
